tools/bazel-tools-tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
```

Comment:
this should give you a warning (if your rules_typescript is recent)
--module is controlled by Bazel

Reply:
Removed.